### PR TITLE
fix the missing link to skip csrf token for openapi

### PR DESCRIPTION
**Pull request by jjtang1985**
## What Has Changed?

This PR will fix the [missing link](https://sap.github.io/cloud-sdk/docs/js/features/openapi/execute-openapi-request/#disabling-csrf-token-fetching).

## Manual Checks?

- [x] Text adheres to the style guide (`vale docs/`)
  - Every sentence is on its own line
  - Headings use [title capitalization](https://capitalizemytitle.com/style/AP/#) (applies also to the sidebar and title of a document)
  - You followed [naming center](https://www.sapbrandtools.com/naming-center/#/dashboard) guidelines when referring to SAP products (e.g. SAP S/4HANA)
- [x] You checked your spelling and grammar (consider using Grammarly, see `CONTRIBUTING.md`)
- [x] You formatted all changed files with prettier (`npm run prettier`)
- [x] You tested if the documentation still builds (`npm run build`)
- [ ] You verified all new and changed links still work (changing the `id` or name of a file can break links)
- [ ] You have updated the [feature matrix](https://github.com/SAP/cloud-sdk/blob/main/docs/components/data/features.js) if you add documentation on a new feature or otherwise required.

### CODE REVIEW

File: docs/js/features/odata/odata-v4-client.mdx [hunk 1]
```diff
@@ -13,6 +13,7 @@ keywords:
   - consume
 ---
 
+import { SkipCsrf } from '../common';
```
Comment (FrankEssenberger) [inline docs/js/features/odata/odata-v4-client.mdx:16]: why split in two imports?

**Review by FrankEssenberger — APPROVED** [review]
LGTM - only one minor comment. Next time consider making a seperate PR for the linting issues.

File: docs/js/features/odata/odata-v4-client.mdx (hunk 1 above)
Comment (jjtang1985) [reply docs/js/features/odata/odata-v4-client.mdx:16]: Because for the examples are different that cannot be shared.

**Comment by jjtang1985** [comment]
> Next time consider making a seperate PR for the linting issues.

@FrankEssenberger @artemkovalyov , I did linting for some files modified by other people multiple times, that made review experience not good. However, running linting is one of the manual checks from the PR template. If we cannot integrate it to the pipeline, people might be worry about running linting for avoiding huge PRs.

**Comment by FrankEssenberger** [comment]
Linting should be in the pipeline so that you only lint what you have changed.

**Pull request merged**